yarn/datadog_checks/yarn/data/conf.yaml.example

```diff
@@ -59,6 +59,28 @@ instances:
     #   <TAG_KEY1>: <YARN_KEY>
     #   <TAG_KEY2>: <YARN_KEY>
 
+    ## @param application_status_mapping - list of key:value elements - optional - default: {RUNNING: ok, FAILED: critical, KILLED: critical}
```

Comment:
Not sure how to specify default values for this @l0k0ms

Reply:
Maybe we can use the example as both default mapping and example:

```
    # application_status_mapping:
    #     ALL: unknown
    #     NEW: ok
    #     NEW_SAVING: ok
    #     SUBMITTED: ok
    #     ACCEPTED: ok
    #     RUNNING: ok
    #     FINISHED: ok
    #     FAILED: critical
    #     KILLED: critical
```